usr/src/uts/common/fs/zfs/zvol.c

```diff
@@ -1058,54 +1058,44 @@ zvol_log_write(zvol_state_t *zv, dmu_tx_t *tx, offset_t off, ssize_t resid,
 {
```

Comment:
I wonder if it would be possible to share some code between this function and `zfs_log_write`.
Not necessary for this change, but would be nice to do in the future.